client/landing/stepper/declarative-flow/hosted-site-migration-v2/index.tsx

```diff
@@ -32,12 +50,28 @@ export default {
 		return {
 			submit: ( props?: ProvidedDependencies ) => {
 				const data = {
-					platform: props?.platform || query.get( 'platform' ),
+					// Site Creation Step
```

Comment:
I really would like to improve this data management.

Reply:
I did a refactor trying to make it a little better. Instead of having a big `data` object, I used variables only where it's needed.

https://github.com/Automattic/wp-calypso/pull/93382/commits/05dc80ba99bc2c43c717c5d77f25953c80120dd6